task_11/app/index.html

```diff
@@ -0,0 +1,365 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <div id="container">
+
+    <header>
+        <nav>
+          <a class="back button" href="#"><i class="fa fa-long-arrow-left" aria-hidden="true"></i> &#8194; Go back</a>
+          <a class="logo" href="/"><img src="img/logo.png" alt="logo"></a>
+          <a class="sign button" href="#"><i class="fa fa-user" aria-hidden="true"></i> &#8194; Sign in</a>
+        </nav>
+    </header>
+
+    <main>
+      <section class="order-section">
+        <section class="order-summary left">
+          <h3 class="order-title"><i class="fa fa-file" aria-hidden="true"></i>
+            Your order
+          </h3>
+          <table>
+            <tr>
+              <th>Items</th>
+              <th>Details</th>
+              <th>Price</th>
+              <th>Delivery</th>
+            </tr>
+            <tr>
+              <td>2 documents</td>
+              <td>2 languages</td>
+              <td value='USD:696.00'>$ 696.00</td>
+              <td>48 hours*</td>
+            </tr>
+          </table>
+          <div class="actions-right">
+              <button class="btn-action button" type="submit">Place order</a>
+              <button class="button">Print a quote</a>
+          </div>
+        </section>
+        <section class="address right">
+            <h3 class="section-title">
+                <i class="fa fa-life-ring icon-offset" aria-hidden="true"></i>
+                Help center
+              </h3>
+            <address>
+                <p>If you need assistance while placing your order, contact one of our beloved Project Managers by phone or email.</p>
+                <a href="tel:+971(0)555 555">+971(0)555 555</a>
```

Comment:
Space in `href` is invalid, but validator should tell you that. I'll skip the rest of the obvious ones :)